cmd/init.go

```diff
@@ -181,9 +181,11 @@ func (h *Handler) Init(ctx context.Context, req *entity.CommandRequest) error {
 		return h.Link(ctx, req)
 	}
 
-	isLoggedIn, _ := h.ctrl.IsLoggedIn(ctx)
-
-	if !isLoggedIn {
+	// Since init can be called by guests, ensure we can fetch a user first before calling. This prevents
+	// us accidentally creating a temporary (guest) project if we have a token locally but our remote
+	// session was deleted.
+	_, err := h.ctrl.GetUser(ctx)
```

Comment:
From reading this code, it looks like we're blocking init for all users no?

Just sanity checking logic here

Reply:
If this returns a user correctly, `err == nil` and we continue on our merry way

Reply:
Oh right cause this is the gateway got it